packages/typespec-azure-resource-manager/test/rules/core-operations.test.ts

```diff
@@ -179,6 +179,7 @@ describe("typespec-azure-resource-manager: core operations rule", () => {
           }
     
           @armResourceOperations
+          #suppress "deprecated" "test"
```

Comment:
It doesn't look like there are any deprecations in the operation template for ArmResourceDeleteAsync.

Reply:
just click the update PR button